packages/editor/README.md

```diff
@@ -440,6 +440,17 @@ getDerivedStateFromError is used to render a fallback UI after an error has been
 
 > **Deprecated** since 5.3, use `wp.blockEditor.getFontSizeClass` instead.
 
+### getTemplateInfo
```

Comment:
If this is a private API, why the doc is being added here. It might be that it's exported publicly as well.

Reply:
Good catch! Fixed with 1f47713e10a01561fc78f212a73bfb9f209b6461